docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,23 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the [requirements](#requirements), it's possible to hotplug an interface by migrating the VM.
```

Comment:
The requirements include "Enabling network interface hotplug support", migration-based hotplug requires the feature gate, please add the explicit requirements that are not needed for migration-based hot-plug.

Reply:
Done